deb/openmediavault/var/www/openmediavault/rpc/session.inc

```diff
@@ -35,6 +35,23 @@ class OMVRpcServiceSession extends \OMV\Rpc\ServiceAbstract {
 		$this->registerMethod("logout");
 	}
 
+	/**
+	* Indicate if user is an administrator (member of group openmediavault-webgui)
+	* @param username The name of the user.
+	* @return TRUE is user is administrator, FALSE otherwise.
+	*/
+	private function isAdmin($username) {
```

Comment:
Try to avoid using the product name in any message (because of branding issues).

I'm asking myself why it should be necessary to test the result of `posix_getgrnam`? I've not tested the code, but are you sure this works here? `nginx/php-fpm` does not run with the correct privileges to access the user database, this must be done in `omv-engined` which has the correct privileges.

The whole function can be simplified (under the assumption `nginx/php-fpm` has the right privileges):
```
    $group = posix_getgrnam("openmediavault-webgui");
    $adminList = $group ? $group['members'] : [ "admin" ];
    return in_array($username, $adminList);
```

Reply:
Just for information, on my system, `posix_getgrnam` and this changes works well (it has been tested before I pushed it) as the group reading access is for accessible to everyone.
I have implemented the simpliest way but maybe not the cleanest, you would do better than me for this :)